qemu/tests/cfg/viorng_in_use.cfg

```diff
@@ -3,32 +3,37 @@
     start_vm = yes
     kill_vm_on_error = yes
     check_guest_bsod = yes
-    login_timeout = 240
+    login_timeout = 360
     suppress_exception = no
     run_bgstress = rng_bat
-    session_cmd_timeout = 240
-    read_rng_cmd = "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
-    rng_data_rex = "0x\w"
-    driver_name = "viorng"
+    session_cmd_timeout = 360
     cdrom_cd1 = isos/windows/winutils.iso
     no no_virtio_rng
-    only Windows
+    Windows:
+        driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
+        target_process = random\w*.exe
+        rng_data_rex = "0x\w"
+        driver_name = "viorng"
+        read_rng_cmd = for /l %i in (1, 1, 1000) do "WIN_UTILS:\random_%PROCESSOR_ARCHITECTURE%.exe"
         i386:
             driver_id_cmd = 'WIN_UTILS:\devcon\wxp_x86\devcon.exe find * | find "VirtIO"'
             driver_check_cmd = "WIN_UTILS:\devcon\wxp_x86\devcon.exe status @DRIVER_ID"
-            driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
         x86_64:
             driver_id_cmd = 'WIN_UTILS:\devcon\wnet_amd64\devcon.exe find * | find "VirtIO"'
             driver_check_cmd = "WIN_UTILS:\devcon\wnet_amd64\devcon.exe status @DRIVER_ID"
-            driver_id_pattern = "(.*?):.*?VirtIO RNG Device"
+    Linux:
+            read_rng_cmd  = "dd if=/dev/random bs=10 count=1000000 2>/dev/null|hexdump"
```

Comment:
1.the rng backend has been change from /dev/random to /dev/urandom,is it still be an issue?
2.if the read_rng_cmd takes too short,the background test check will fail.